suite-native/atoms/src/Button/Button.tsx

```diff
@@ -193,7 +201,7 @@ export const buttonSchemeToColorsMap = {
     },
 } as const satisfies Record<ButtonColorScheme, ButtonColorSchemeColors>;
 
-const sizeToDimensionsMap = {
+export const sizeToDimensionsMap = {
```

Comment:
Since this is now accessible everywhere where is the `atoms` package imported, could you please make the name more specific? Because this import might cause some confusion (it is not clear "size of what" is mapped in this object):

```
import { sizeToDimensionsMap } from '@suite-native/atoms';
```

```suggestion
export const buttonSizeToDimensionsMap = {
```

Reply:
https://github.com/trezor/trezor-suite/pull/17981/commits/47c932e7e017d515a6346a59d9ceca1f2d6292cd good point